packages/react-components/src/trace-explorer/trace-explorer-views-widget.tsx

```diff
@@ -177,14 +177,9 @@ export class ReactAvailableViewsWidget extends React.Component<ReactAvailableVie
             outputDescriptors.push(...descriptors);
         }
         /**
-         * Remove outputs of type DATA_TREE since the view is not supported in theia-traceviewer
+         * Remove outputs of type DATA_TREE until the tooltip is implemented
```

Comment:
Doesn't the tooltip comment apply to the scatter chart? I guess this comment for data tree should not be touched?

Reply:
Fixed.